### DISCUSSION TITLE
New Windows User - It's running but new jobs are not starting - config help

### DISCUSSION THREAD

**Discussion by MajorTomG** [swar/Swar-Chia-Plot-Manager#742] [Q&A]
I've got Chia 1.1.6 installed and that's been plotting fine for a few days.

Just installed Python 3.9.5 followed the setup, configured my config.yaml (attached) but when I run and view the manager, no plotting happens. See below. Temp folder remains empty.

![image](https://user-images.githubusercontent.com/79742292/119225057-7e172080-baf9-11eb-9aab-ddf4a71ea711.png)

I followed the guide posted [here](https://www.youtube.com/watch?v=Wl6FVRAj86I) initially and then went over the instructions posted on the main git page. Log analysis here if it's any help.

![image](https://user-images.githubusercontent.com/79742292/119225137-f2ea5a80-baf9-11eb-9708-e28b7e512984.png)

[config.zip](https://github.com/swar/Swar-Chia-Plot-Manager/files/6526312/config.zip)

IF it matters, my commands start with "python" rather than "python3".

**Comment by lixcxc96** [comment]
> 
![20210522201813](https://user-images.githubusercontent.com/84433255/119226273-df131880-bb3a-11eb-8eb1-641213683107.png)

**Reply by MajorTomG** [reply]
Thanks,

Just tried this with a mapped, lettered drive but still the same thing. Any other suggestions?

**Comment by nourcosarl** [comment]
Exact same problem here running 1.1.6
I must add that Swar plotter log files generating logs with following:
"Second instance. Quitting."

**Reply by MajorTomG** [reply]
Do let me know if you get anywhere with this. I can't figure it out at all.
